Mathlib/Algebra/Order/Floor/Defs.lean

```diff
@@ -1191,9 +974,6 @@ theorem ceil_eq_zero_iff : ⌈a⌉ = 0 ↔ a ∈ Ioc (-1 : α) 0 := by simp [cei
 theorem ceil_eq_on_Ioc (z : ℤ) : ∀ a ∈ Set.Ioc (z - 1 : α) z, ⌈a⌉ = z := fun _ ⟨h₀, h₁⟩ =>
   ceil_eq_iff.mpr ⟨h₀, h₁⟩
 
-theorem ceil_eq_on_Ioc' (z : ℤ) : ∀ a ∈ Set.Ioc (z - 1 : α) z, (⌈a⌉ : α) = z := fun a ha =>
-  mod_cast ceil_eq_on_Ioc z a ha
-
```

Comment:
This splitting is somewhat surprising to me - I'd expect `ceil_eq_on_Ioc` and `ceil_eq_on_Ioc'` to be in the same file... Can we add something to the docstring of the two files to distinguish what should go where?

Reply:
I've postponed `ceil_eq_on_Ioc` so it can sit with its primed analogue now.